ccp/docker-compose.yml

```diff
@@ -31,6 +31,9 @@ services:
       BEAM_PROXY_URL: http://beam-proxy:8081
       RETRY_COUNT: ${FOCUS_RETRY_COUNT}
       EPSILON: 0.28
+      QUERIES_TO_CACHE: '/queries_to_cache.conf'
+    volumes:
+      - ./queries_to_cache.conf:/queries_to_cache.conf
```

Comment:
I am a bit skeptical of the relativ path. If it works it works but in most places we hardcoded `/srv/docker/bridgehead`

Reply:
thx, changed